test/ecdh_test.c

```diff
@@ -326,6 +326,15 @@ int main(int argc, char **argv)
 
 	/* Iterate over curves */
 	for (i = 0; i < NUM_ECDH_TESTS; i++) {
+
```

Comment:
Remove blank line

Reply:
You are right, according to coding standards the blank lines are superfluous. I added them intentionally to make the code better readable. I'll check after this pr if I can probably make the loops smaller by putting code into subroutines. There are many places where test routines get lengthy ...